frontend/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
     "presets": ["next/babel"],
-    "plugins": []
+    "plugins": ["@shopify/web-worker/babel"]
```

Comment:
You add `@shopify/web-worker/babel` as a plugin for all files. Will it only apply itself to workers? I feel like we should be selectively loading this plugin for `.worker.[jt]sx?` files only.

Reply:
Based on [this](https://www.npmjs.com/package/@shopify/web-worker), it seems the Babel plugin is only used to collate and process calls to `createWorkerFactory`, and so needs to process all source files.